services/editor/src/services/search-service.js

```diff
@@ -22,7 +39,10 @@ const createSearchFunction = endpoint =>
         credentials: 'same-origin',
       },
     );
-    return await response.json();
+
+    const results = await response.json();
+
+    return endpoint === 'search' ? results : filterInternalKeys(results);
```

Comment:
maybe just modify `getSuggestions` to also use `filterInternalKeys`

Reply:
good point, fixed.